draft-packages/badge/KaizenDraft/Badge/styles.module.scss

```diff
@@ -24,22 +21,54 @@ $small: $ca-grid;
   text-align: center;
 }
 
+.badge.large {
+  display: inline-flex;
+  justify-content: center;
+  border-radius: $spacing-xl;
+  font-size: $typography-data-medium-font-size;
+  line-height: $typography-data-medium-line-height;
+  letter-spacing: $typography-data-medium-letter-spacing;
+  padding: 2px 1.875rem;
+  width: 24px;
+}
+
 .default {
   background-color: $dt-color-background-default;
   color: $color-purple-800;
 }
 
-.reversed.active,
 .active {
   background-color: $color-blue-500;
   color: $color-white;
 }
 
+.reversed.active {
+  background-color: $color-green-300;
+  color: $color-purple-800;
+}
+
+//TODO remove
```

Comment:
Unsure what this comments means here: did you forget to remove this for this PR? Or will we need to remove this in the future? If the latter, please add a bit more info... same applies to a bit further down on line 71.